aws.go

```diff
@@ -6,17 +6,58 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/percona/rds_exporter/config"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-	PeriodSeconds = 60
-	DelaySeconds  = 600
-	RangeSeconds  = 600
+	Period = 60 * time.Second
+	Delay  = 600 * time.Second
+	Range  = 600 * time.Second
+
+	latencyDesc = prometheus.NewDesc(
+		"rds_latency",
+		"The difference between the current time and timestamp in the metric itself",
+		[]string{"instance", "region"},
+		map[string]string(nil),
+	)
 )
 
+type Scrape struct {
```

Comment:
Probably `Scraper`?

Reply:
I would prefer `Scrape` since `Scraper` would denote an interface.

Reply:
Well, that's an object type, a "thing", a noun, not a method/verb. Also, it has a method Scrape. Scrape does not scrape, but scraper scrapes – that's what it does. :)

Honestly, I would prefer a function there – but that's another topic.

Reply:
Changed in https://github.com/percona/rds_exporter/pull/4